elearning-backend/app/Http/Controllers/API/SessionsController.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+namespace App\Http\Controllers\API;
+
+use App\Http\Controllers\Controller;
+use App\Http\Requests\UserFieldPostRequest;
+use App\Models\User;
+use Illuminate\Http\Request;
+
+class SessionsController extends Controller
+{
+    public function login(Request $request)
+    {
+        $validator = UserFieldPostRequest::createFrom($request);
```

Comment:
```suggestion
    public function login(UserFieldPostRequest $request)
    {
```
You can do this instead, pls also apply it to other controller methods